doc/specs/stdlib_io.md

```diff
@@ -305,3 +305,38 @@ Exceptions trigger an `error stop` unless the optional `err` argument is provide
 {!example/io/example_get_file.f90!}
 ```
 
+## `print_array` - Print an array to an output unit
```

Comment:
Minor comment: the name could be discussed with the community. Personally, I would prefer `disp`.